scripts/pi-hole/php/header.php

```diff
@@ -433,7 +445,7 @@ function pidofFTL()
                     </li>
                     <li<?php if($scriptname === "db_queries.php"){ ?> class="active"<?php } ?>>
                         <a href="db_queries.php">
-                            <i class="fa fa-fw menu-icon fa-file-alt"></i> Query Log
+                            <i class="fa fa-fwDNS Control menu-icon fa-file-alt"></i> Query Log
```

Comment:
I'm not sure why you added "DNS Control" classes here, but i think **you forgot the space**:
```suggestion
                            <i class="fa fa-fw DNS Control menu-icon fa-file-alt"></i> Query Log
```

Or remove these extra classes, if it was a mistake:
```suggestion
                            <i class="fa fa-fw menu-icon fa-file-alt"></i> Query Log
```

Reply:
Clearly a copy&pasta in the wrong spot.